conda_build_all/builder.py

```diff
@@ -84,7 +84,13 @@ def list_metas(directory, max_depth=0, config=None):
 
         if 'meta.yaml' in files:
             if hasattr(conda_build, 'api'):
-                packages.append(conda_build.api.render(new_root, config=config)[0])
+                pkgs = conda_build.api.render(new_root, config=config,
+                                        finalize=False, bypass_env_check=True)
+                if hasattr(pkgs[0], 'config'):
+                    pkgs = [pkgs[0]]
+                else:
+                    pkgs = [pkg[0] for pkg in pkgs]
+                packages.extend(pkgs)
```

Comment:
Sorry I think the subtlety of this is escaping me. What are we solving here? I'm sure there is a reason for it just unclear on what it is.

Reply:
cb2 returns just one tuple: ``(metadata, needs_download, needs_render_in_build_env)``
cb3 returns a list of those tuples

In either case, all we care about is the first entry of the tuples.